data/json/mapgen/office_dentist.json

```
@@ -0,0 +1,169 @@
+[
+  {
+    "type": "mapgen",
+	"method": "json",
+	"om_terrain": [ "office_dentist" ],
+	"weight": 500,
+	"object": {
+		"fill_ter": "t_region_groundcover_urban",
+		"rows": [
+		  "   sy----y----y----ys   ",
+		  " ! sy----y----y----ys ! ",
+		  " ! sy----y----y----ys ! ",
+		  " ! sy----y----y----ys ! ",
+		  " ! sy----y----y----ys ! ",
+		  " ! ssssssssssssssssss ! ",
+		  "   ##w#####dd########   ",
+		  "  *#DPD#p..........b#*  ",
+		  "  *w.h.d...DPDff...bw*  ",
+		  "  *#BBB#....h.ff...bw*  ",
+		  "   #####...#####...b#*  ",
+		  "  *#MSR#..........p##   ",
+		  "  *w,,,d..##d##d####+   ",
+		  "  *#cnc#..#S,#S,#X%#    ",
+		  "  s#####..#t&#t&#,,#*   ",
+		  "  sd......#######|^#*   ",
+		  "  s##d##...........w*   ",
+		  "   #L__##d###d###d##* [ ",
+		  "   #L_=#S,E#S,E#S,E#*   ",
+		  "   #LWF#C,,#C,,#C,,#    ",
+		  "   ######w###w###w##    ",
+		  "    *** *** *** ***     ",
+		  "  [                     ",
+		  "                        "
+		],
+		"terrain": {
+			" ": "t_region_groundcover_urban",
+			".": "t_floor",
+			"#": "t_wall_w",
+			"C": "t_linoleum_white",
+			"S": "t_linoleum_white",
+			"w": "t_window",
+			"t": "t_linoleum_white",
+			"d": "t_door_c",

```

```suggestion
			"d": [ [ "t_door_c", 2 ], [ "t_door_o", 1 ] ],
```
Another bit of personal preference but having variation in open and closed doors helps make the map feel more affected by the chaos.